src/test/java/Entities/RecipeTest.java

```diff
@@ -0,0 +1,53 @@
+package test.java.Entities;
+import main.java.Entities.Recipe;
+import main.java.Entities.Item;
+import main.java.Entities.Ingredient;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Collections;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class RecipeTest {
+    @Test
+    public void testName() {
+        Recipe recipe = new Recipe("name", "description", "instructions", Collections.emptyList());
+        assertEquals("name", recipe.name());
+    }
+
+    @Test
+    public void testDescription() {
+        Recipe recipe = new Recipe("name", "description", "instructions", Collections.emptyList());
+        assertEquals("description", recipe.description());
+    }
+
+    @Test
+    public void testInstructions() {
+        Recipe recipe = new Recipe("name", "description", "instructions", Collections.emptyList());
+        assertEquals("instructions", recipe.instructions());
+    }
```

Comment:
If you want, you can declare a `BeforeTest`  function which is run before each test. In this you can create such a shared recipe object to use within each test. Remember, `RecipeTest` is simply a class, it can have instance variables just like any other. You should be able to find more information online. Also the Jshell has similar in it's test.